### ISSUE TITLE
Init has errors in latest version of deno

### ISSUE THREAD

**Issue by justinmchase** [dylanpyle/version#1]
I just ran this
```sh
deno run -A https://deno.land/x/version/index.ts init
```

And got these typescript errors:
```sh
Download https://deno.land/x/version/index.ts
Warning Implicitly using latest version (1.0.2) for https://deno.land/x/version/index.ts
Download https://deno.land/x/version@1.0.2/index.ts
Download https://deno.land/x/version@1.0.2/git.ts
Download https://deno.land/x/semver@v1.0.0/mod.ts
Download https://deno.land/x/version@1.0.2/user-error.ts
Check https://deno.land/x/version/index.ts
error: TS2339 [ERROR]: Property 'operator' does not exist on type 'never'.
    if (high!.operator === comp || high!.operator === ecomp) {
              ~~~~~~~~
    at https://deno.land/x/semver@v1.0.0/mod.ts:1744:15

TS2339 [ERROR]: Property 'operator' does not exist on type 'never'.
    if (high!.operator === comp || high!.operator === ecomp) {
                                         ~~~~~~~~
    at https://deno.land/x/semver@v1.0.0/mod.ts:1744:42

TS2339 [ERROR]: Property 'operator' does not exist on type 'never'.
      (!low!.operator || low!.operator === comp) &&
             ~~~~~~~~
    at https://deno.land/x/semver@v1.0.0/mod.ts:1751:14

TS2339 [ERROR]: Property 'operator' does not exist on type 'never'.
      (!low!.operator || low!.operator === comp) &&
                              ~~~~~~~~
    at https://deno.land/x/semver@v1.0.0/mod.ts:1751:31

TS2339 [ERROR]: Property 'semver' does not exist on type 'never'.
      ltefn(version, low!.semver)
                          ~~~~~~
    at https://deno.land/x/semver@v1.0.0/mod.ts:1752:27

TS2339 [ERROR]: Property 'operator' does not exist on type 'never'.
    } else if (low!.operator === ecomp && ltfn(version, low!.semver)) {
                    ~~~~~~~~
    at https://deno.land/x/semver@v1.0.0/mod.ts:1755:21

TS2339 [ERROR]: Property 'semver' does not exist on type 'never'.
    } else if (low!.operator === ecomp && ltfn(version, low!.semver)) {
                                                             ~~~~~~
    at https://deno.land/x/semver@v1.0.0/mod.ts:1755:62

Found 7 errors.
```

**Comment by dylanpyle** [comment]
Ooh, nice catch. Looks like potentially this was fixed in the latest version of [semver](https://github.com/justjavac/deno-semver), and I imagine the upgrade is smooth too - I'll take a look shortly

**Comment by justinmchase** [comment]
https://github.com/dylanpyle/version/pull/2

**Comment by justinmchase** [comment]
Yeah it turns out just bumping the version from `1.0.0` to `1.4.0` fixes this issue.

**Comment by dylanpyle** [comment]
Published the fix in [1.0.3](https://deno.land/x/version@1.0.3), thank you!

(only considered after the fact that `get` probably justified a minor release, but 🤷 )
